.github/workflows/build_images.yml

```diff
@@ -36,20 +36,22 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3
-      - name: Login to DockerHub
+      - name: Login to ghcr.io
         uses: docker/login-action@v2 
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build docker image and push to dockerhub
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
I'm not sure if this is correct, in my workflows I set it to the name of the organization which owns the repo

Reply:
good point. I'll see after the merge and adjust accordingly.

Reply:
seems like it worked :)